README.md

```diff
@@ -72,6 +74,39 @@ cf set-env $YOUR_APP_NAME LOGS_CONFIG '[{"type":"tcp","port":"10514","source":"j
 #### General configuration of the Datadoc Agent
 All the options supported by the Agent in the main configuration file (`lib/dist/datadog.yaml`) can also be set through environment variables as described in the [documentation of the Agent](https://github.com/DataDog/datadog-agent/blob/master/docs/agent/config.md#environment-variables).
 
+#### .NET Traces
```

Comment:
🙇 👏

Reply:
Can't take the credit, mostly comes from the great work @zippolyte did - https://github.com/DataDog/datadog-cloudfoundry-buildpack/pull/61/files#diff-04c6e90faac2675aa89e2176d2eec7d8R82

Reply:
🙇